enterprise/views.py

```diff
@@ -1786,7 +1785,10 @@ def get(self, request, enterprise_uuid, course_id):
         """
         # Check to see if access to the course run is restricted for this user.
         embargo_url = EmbargoApiClient.redirect_if_blocked(
-            [course_id], request.user, get_client_ip(request)[0], request.path)
+            request=request,
+            course_run_ids=[course_id],
```

Comment:
[clarification] Does `redirect_if_blocked` accept `course_run_ids`? It looks like it only accepts a single `course_key` kwarg?

```python
def redirect_if_blocked(
    request: Request,
    course_key: CourseKey,
    access_point: str = 'enrollment',
    user: Optional[types.User] = None,
) -> Optional[str]:
```

Reply:
phew, good catch.
I should try to manually test this.

Reply:
ah, wait, so - `EmbargoApiClient.redirect_if_blocked()` is really just a wrapper around the `embargo_api.redirect_if_blocked()` function.  If you look at the change in `lms.py`, we're using the `course_key` argument - our wrapper calls the real function in a loop, one iteration per course run id.

Reply:
Oh, yeah, my bad! The similar names of these methods got me conflated :)